.travis.yml

```diff
@@ -3,14 +3,20 @@ python: 2.7
 sudo: false
 env:
   matrix:
-  - TOX_ENV=py26
-  - TOX_ENV=py27
-  - TOX_ENV=py33
-  - TOX_ENV=py34
-  - TOX_ENV=pypy
-  - TOX_ENV=docs
-  - TOX_ENV=system-tests
-  - TOX_ENV=system-tests3
+  # - TOX_ENV=py26
+  # - TOX_ENV=py27
+  # - TOX_ENV=py33
+  # - TOX_ENV=py34
+  # - TOX_ENV=pypy
+  # - TOX_ENV=docs
+  # - TOX_ENV=system-tests
+  # - TOX_ENV=system-tests3
+  - TOX_ENV=gae
+  global:
+  - GAE_PYTHONPATH=$HOME/.cache/google_appengine
```

Comment:
RE: source for curlies, see [this answer][1]. In particular

> Doing this everywhere, instead of just in potentially ambiguous cases, can be considered good programming practice. This is both for consistency and to avoid surprises like `$foo_$bar.jpg`, where it's not visually obvious that the underscore becomes part of the variable name.

[1]: http://stackoverflow.com/a/8748880/1068170